test/Microsoft.AspNetCore.Mvc.DataAnnotations.Test/Internal/DataAnnotationsMetadataProviderTest.cs

```diff
@@ -977,6 +977,36 @@ public void CreateDisplayMetadata_DisplayName_LocalizeWithStringLocalizer()
         }
 
         [Fact]
+        public void CreateDisplayMetadata_EnumGroupedDisplayNamesAndValues_ReflectsDisplayAttributeOrder()
+        {
+            // Arrange
+            var expectedKeyValuePairs = new List<KeyValuePair<EnumGroupAndName, string>>
+            {
+                new KeyValuePair<EnumGroupAndName, string>(new EnumGroupAndName(string.Empty, nameof(EnumWithDisplayOrder.Null)), "3"),
```

Comment:
Given that the `Null` enum member doesn't have the `DisplayAttribute` shouldn't its order be the default value of 1000 and thus be at the end of the list?